src/common/TrackedOp.h

```diff
@@ -312,13 +315,13 @@ class TrackedOp : public boost::intrusive::list_base_hook<> {
   }
 
   void mark_tracepoint(std::string_view name,
-                       std::map<const char*, int64_t> kvs) {
+                       std::initializer_list<
```

Comment:
@yangdongsheng the build stuff for jaeger is pretty sensitive, have you tested `std::initializer_list<std::pair<string_view, Value>>` instead of using `std::pair<opentracing::string_view,pentracing::Value>>` ? 
In that case we don't need cmake changes